src/mito2/src/flush.rs

```diff
@@ -382,6 +395,7 @@ impl RegionFlushTask {
                 index_file_size: sst_info.index_metadata.file_size,
                 num_rows: sst_info.num_rows as u64,
                 num_row_groups: sst_info.num_row_groups,
+                max_sequence: max_sequence.unwrap_or(0),
```

Comment:
We can get the max sequence from this memtable.
```suggestion
                max_sequence: max_sequence.unwrap_or(0),
```